### ISSUE TITLE
how to config resolving

### ISSUE THREAD

**Issue by craigcosmo** [benmosher/eslint-plugin-import#734]
**project structure**
```
home_folder
    | home.js
subpage_folder
    | contact.js
app.css
app.js
```

**app.js**

```js
import home from 'home'
import contact from 'contact'
import 'app.css'
```

What I expect: no error

But eslint result: can not resolve home, contact and app

What should I do in the eslint config so that eslint result will meet my expectation, in other word, eslint will understand there are file like `home`, `contact`, `app` and it should not produce error?

**Comment by k15a** [comment]
Your code should be:

```JS
import home from './home_folder/home'
import contact from './subpage_folder/contact'
import './app.css'
```

**Comment by craigcosmo** [comment]
webpack's resolve allow us to do code like my example

webpack.config.js (webpack v2)
```js
entry:...,
output:{...},
resolve: {
		modules: ['home_folder', 'subpage_folder ', 'node_modules']
	},
```

so our code

```js
import home from 'home'
import contact from 'contact'
import 'app.css'
```

would run just fine. Except when we eslint it.

**Comment by k15a** [comment]
Then you need https://www.npmjs.com/package/eslint-import-resolver-webpack.

**Comment by craigcosmo** [comment]
Hey that works great, thanks. `npm run lint` now will not give me any error.

Do you know how get this resolver to work with sublime?

In sublime I got a bunch of error 

<img width="576" alt="screen shot 2017-02-03 at 4 17 14 pm" src="https://cloud.githubusercontent.com/assets/538657/22585939/dd89c6d2-ea2c-11e6-8a25-e5f62bdb12fd.png">


<img width="561" alt="screen shot 2017-02-03 at 4 21 58 pm" src="https://cloud.githubusercontent.com/assets/538657/22585950/eb9d3754-ea2c-11e6-8be3-1add5e9e0f53.png">

**Comment by k15a** [comment]
https://github.com/benmosher/eslint-plugin-import#sublimelinter-eslint

**Comment by craigcosmo** [comment]
```
{
    "folders":
    [
        {
            "path": "code"
        }
    ],
    "SublimeLinter":
    {
        "linters":
        {
            "eslint":
            {
                "chdir": "${project}/code"
            }
        }
    }
}
```

do you know what does `path:"code"` mean?

And what is the ${project}?

**Comment by k15a** [comment]
Don't know. For me it's working without this in sublime.

**Comment by craigcosmo** [comment]
strange, do you also use the webpack module resolve?

**Comment by k15a** [comment]
In one project yes. Do you use sublime 2 or 3? And on which OS?

**Comment by craigcosmo** [comment]
sublime 3, mac siera

**Comment by craigcosmo** [comment]
the sublime package I'm using

<img width="357" alt="screen shot 2017-02-03 at 10 06 38 pm" src="https://cloud.githubusercontent.com/assets/538657/22596164/11c03c1c-ea5d-11e6-9fb1-daa248c13681.png">

**Comment by k15a** [comment]
Using sublime 3 and sierra as well. Strange that this is not happening to me.

**Comment by craigcosmo** [comment]
my config

```json
{
	"parserOptions": {
		"ecmaFeatures": {
			"jsx": true,
			"modules": true
		}
	},
	"globals": {
		"localStorage": true,
		"google": true,
		"process": true,
		"__dirname":true,
		"describe":true,
		"require":true,
		"module":true,
		"it":true
	},
	"parser": "babel-eslint",
	"env": {
		"browser": true,
		"es6": true,
		"node": true
	},
	"settings":{
		"import/resolver": {
			"webpack": {
				"config": "webpack.config.babel.js"
			}
		}
	},
	  
	"rules": {
		"quote-props": ["off", "as-needed"],
		"no-unused-vars": ["off"],
		"no-mixed-spaces-and-tabs": "warn",
		"import/no-unresolved": "error"
	},
	"plugins": [
		"react",
		"import"
	]
}
```

**Comment by pseidemann** [comment]
hey @craigcosmo could you ever solve this issue? I have the same problem

**Comment by craigcosmo** [comment]
@pseidemann no. But i got it work without other git

https://github.com/tleunen/eslint-import-resolver-babel-module
and this
https://github.com/tleunen/babel-plugin-module-resolver

download the latest beta of both, it will work

**Comment by ljharb** [comment]
You need to add the node resolver, I suspect.
